src/adapter-nextjs/page-props/category.ts

```diff
@@ -0,0 +1,74 @@
+import type { ExtraStoryFields, Story } from '@prezly/sdk';
+import type { GetServerSidePropsContext, GetServerSidePropsResult } from 'next';
+
+import { DEFAULT_PAGE_SIZE } from '../../data-fetching';
+import type { PaginationProps } from '../../infinite-loading/types';
+import { getNewsroomServerSideProps } from '../getNewsroomServerSideProps';
+import { processRequest } from '../processRequest';
+
+import type { PropsFunction } from './lib/types';
+
+export interface CategoryPageProps<StoryType extends Story = Story> {
+    stories: StoryType[];
+    pagination: PaginationProps;
+}
+
+interface Options {
+    storiesIncludeFields?: (keyof ExtraStoryFields)[];
```

Comment:
Why not just call this the same as the interface - `extraStoryFields`? Seems more readable.

Reply:
Yeah, that makes much more sense! Thank you!